src/compiler/emitter.ts

```diff
@@ -4215,15 +4219,19 @@ var __param = this.__param || function(index, decorator) { return function (targ
                     return emitOnlyPinnedOrTripleSlashComments(node);
                 }
 
-                emitStart(node);
-                if (isES6ExportedDeclaration(node)) {
-                    write("export ");
+                if (!isModuleMergedWithES6Class(node)) {
+                    emitStart(node);
+                    if (isES6ExportedDeclaration(node)) {
+                        write("export ");
+                    }
+
+                    write("var ");

```

Comment:
`var` and `let` declarations are both mutable bindings in the environment record and exported names, `const` declarations are immutable bindings (obviously). Import bindings are immutable as well, but are an indirect binding to the binding in the environment record of the imported module.

* [8.1.1.1.2 - CreateMutableBinding](http://people.mozilla.org/~jorendorff/es6-draft.html#sec-declarative-environment-records-createmutablebinding-n-d)
* [8.1.1.1.3 - CreateImmutableBinding](http://people.mozilla.org/~jorendorff/es6-draft.html#sec-declarative-environment-records-createimmutablebinding-n-s)
* [8.1.1.1.5 - SetMutableBinding](http://people.mozilla.org/~jorendorff/es6-draft.html#sec-declarative-environment-records-setmutablebinding-n-v-s)
* [8.1.1.5 - Module Environment Records](http://people.mozilla.org/~jorendorff/es6-draft.html#sec-module-environment-records)
* [8.1.1.5.5 - CreateImportBinding](http://people.mozilla.org/~jorendorff/es6-draft.html#sec-createimportbinding)
* [15.2.3.3 - Static Semantics: ExportedBindings](http://people.mozilla.org/~jorendorff/es6-draft.html#sec-exports-static-semantics-exportedbindings)